extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -56,9 +57,17 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     this.injector = injector;
     this.gmailVariant = gmailVariant;
     this.notifications = notifications;
+    this.webmailCommon = new WebmailCommon(acctEmail, injector);
+  }
+
+  getIntervalFunctions = (): Array<IntervalFunction> => {
+    return [
+      { interval: 1000, handler: this.everything },
+      { interval: 30000, handler: () => this.webmailCommon.addOrRemoveEndSessionBtnIfNeeded() }
```

Comment:
You should keep the same callback style if possible, eg
```js
      { interval: 1000, handler: () => this.everything() },
      { interval: 30000, handler: () => this.webmailCommon.addOrRemoveEndSessionBtnIfNeeded() }
```
or
```js
      { interval: 1000, handler: this.everything },
      { interval: 30000, handler: this.webmailCommon.addOrRemoveEndSessionBtnIfNeeded }
```
It confused me for a second. Unless there is a special reason, in which case maybe a comment would be appropriate

Reply:
yes, thanks, I missed that.